SAP2000_Adapter/CRUD/Create/Node.cs

```diff
@@ -96,7 +96,7 @@ private bool SetObject(Node bhNode)
                 }
             }
 
-            if (!bhNode.Orientation.Equals(Basis.XY))
+            if (bhNode.Orientation != null && !bhNode.Orientation.Equals(Basis.XY))
```

Comment:
since this is in SetObject(), which is also used by Update(), we probably want to remove the check that the orientation is not Basis.XY; otherwise if someone creates a node with a funny orientation, and then wants to use update to put it back to global, they won't be able to.

Reply:
However, the second check is never true:
```
bhNode.Orientation.Y == Basis.XY.Y
true
bhNode.Orientation.X == Basis.XY.X
true
bhNode.Orientation.Z == Basis.XY.Z
true
bhNode.Orientation == Basis.XY
false
```

That's from the immediate window after passing in a node with orientation set to the global coordinates.

Reply:
With that, I suggest:
```suggestion
            if (bhNode.Orientation != null)
```

Reply:
I've built and tested that commit, so would be happy to commit myself, or approve on-sight.

Reply:
looking again, I'm seeing that we weren't using the native BHoM equals method- if we use Orientation.IsEqual(Basis.XY) it works. That enables checking for that and setting the orientation with the much simpler `m_model.PointObj.SetLocalAxes(name, 0, 0, 0)` method, which will also mean the SAP model is easier to use after.